src/components/AdvancedSearch/AdvancedSearch.tsx

```diff
@@ -47,14 +50,27 @@ const AdvancedSearch: React.FC<{
 }> = ({ languages: previousLanguages }) => {
   const router = useRouter();
 
-  const [searchQuery, setSearchQuery] = useState<SearchQuery>({
+  const [searchQuery] = useState<SearchQuery>({
     ...SearchQueryDefaults,
   });
 
   const [emptySearchError, setEmptySearchError] = useState(false);
   const [dateRangeError, setDateRangeError] = useState("");
 
   const [languages, setLanguages] = useState<FacetItem[]>([]);
+  const [languageFilters, setLanguageFilters] = useState(
+    findFiltersForField(searchQuery.filters, filterFields.language)
+  );
+  const [queries, setQueries] = useState<Query[]>(searchQuery.queries);
+  const [formatFilters, setFormatFilters] = useState<Filter[]>(
+    findFiltersForField(searchQuery.filters, filterFields.format)
+  );
+  const [startFilter, setStartFilter] = useState<Filter[]>(
```

Comment:
does it have to be a `Filter[]` here?  aren't startFilter and endFilter guaranteed to only be one thing?